oss-internship-2020/libuv/callbacks/callbacks.h

```diff
@@ -0,0 +1,41 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef CALLBACKS_H
+#define CALLBACKS_H
+
+#include <uv.h>
+
+extern "C" {
+
+// idle-basic
+
+void IdleCallback(uv_idle_t *handle);
```

Comment:
Keep the asterisk close to the type. The `.clang-format` in the SAPI root dir enforces this.

Here and elsewhere in this PR.